bin/clover/src/pipelines/azure/spec.ts

```diff
@@ -690,8 +950,8 @@ function parseEndpointPath(path: string) {
   if (!!resourceGroupParam !== !!resourceNameParam) return undefined;
 
   const serviceName = providerNamespace.split(".").pop() || providerNamespace;
-  const capitalizedResource =
-    resourceType.charAt(0).toUpperCase() + resourceType.slice(1);
+  const capitalizedResource = resourceType.charAt(0).toUpperCase() +
```

Comment:
I'm not sure, but I'm starting to worry you and I have dueling typescript formatters :)